client/src/components/Meal.jsx

```diff
@@ -69,15 +71,21 @@ const CalorieText = styled('span', {
 
 export function Meal({ meal }) {
   const [calorieExceeds, setCalorieExceeds] = useState(false)
-  const { parsedDate, calories, name, parsedTime } = meal
-  const { day, month, year } = parsedDate
-  const { hour, minute } = parsedTime
+
+  const date = new Date(meal.date * 1000)
```

Comment:
1000-ზე რატომ ამრავლებ?

Reply:
-->(elapsed seconds since unix epoch) 
ესე ვინახავ ,დეფოლტად ხომ მილისექონდებში ინახავს,აქ ვყოფ 1000ზე რომ სექონდებში იყოს
![unix](https://user-images.githubusercontent.com/68153140/151675756-83e77567-df48-4761-956d-77658285bfd5.png)
ამიტომაც დამჭირდა გამრავლება 1000ზე
თუ არ იყო ამის გაკეთება საჭირო ამოვშლი

Reply:
My bad, seconds კი არა milliseconds-ებშია. 

getTime-ს ლინკი რომ გამოგიგზავნე წაიკითხე?

<img src="https://user-images.githubusercontent.com/41262747/151675974-fc5ec323-e127-4b95-a2b0-aa98183a7e32.png" />

<img src="https://user-images.githubusercontent.com/41262747/151676079-91a0a81e-a6ad-4e22-8043-0015d6fd8c2b.png" />

Reply:
![snipet](https://user-images.githubusercontent.com/68153140/151676380-dd244558-3b88-40c7-82de-a278bb87fed1.png)
date-fns ფუნქციას ვიყენებ იგივე თემა არაა? მაძლევს unix-ს.

ფრონტიდან isoStringi უნდა გავაგზავნო და ბექზე გადავიყვანო უნიქსში? 

გოგა მეორე სურათით რაზე გამოგზავნე,მილისექონდებში რომაა მაგის ჩვენება გინდოდა?

Reply:
მოიცა, არაეფექტურია ახლა ჩვენი საუბარი.

მე გაძლევ ინსტრუქციას/დავალებას — რა არის გასაკეთებელი. ის თუ როგორ მიხვიდე იქამდე, არის შენი გადასაწყვეტი.

* არ ვიცი/არ მახსოვს როგორ მუშაობს date-fns-ს getTime. თუ იმას აკეთებს, რაც დავალებისთვის გჭირდება — კარგია. გადავამოწმებ კოდის რევიუს დროს და არა Q&A-ს დროს
* ISOString გაგზავნო თუ არა, არის იმპლემენტაციის დეტალი, არის შენი გადასაწყვეტი. ამოცანა სწორად რომ გაგეგო, მიხვდებოდი რომ ISOString არ უნდა გაგზავნო. მე ფოტოში უბრალოდ განახე, რომ getTime მილიწამებში გაძლევს ინფორმაციას
* თუ მე დავაკელი დეტალები დავალებას, ვიდრე იმპლემენტაციაზე გადახვალ, დააზუსტე

**დავალებაა შემდეგი:**
* ყველა საკვებს უნდა ჰქონდეს date property, რომელიც არის timestamp (64-bit INT — Unix Time)
* მოცემული property-თ მომხმარებელს უნდა შეეძლოს საკვების გაფილტვრა, თარიღის საწყისიდან საბოლოო მნიშვნელობამდე
* მომხმარებელს შეუძლია მხოლოდ საწყისი ან მხოლოდ საბოლოო თარიღი აირჩიოს

Do your own research. You had enough tips. რომ მორჩები დაფუშე და კოდს დავარევიუებ. თუ რაიმე ხარვეზი იქნება გეტყვი, თუ უკეთესი გზებიც არსებობს — დაგიეფრუვებ, მაგრამ მაგასაც მოგწერ

Reply:
ფაქტიურად ეგ მაქ გაკეთებული , სექონდებში რომ გეწერა literally მივიღე და სექონდებში გადავიყვანე,
რომ გამოგზავნე სურათი ვერ გავიგე მიზეზი ბოლომდე   და ჩაგეკითხე ეგაა და ეგ,

-->date-fns ფუნქციას ვიყენებ იგივე თემა არაა? მაძლევს unix-ს.
უბრალოდ შეგეკითხე,იმპლემენტაციაზე აქცენტი არც მქონდა.

რომ არ გავწელო ხვალ დავპუშავ დილას, დავიძინებ ეხლა